app/controllers/application_controller.rb

```diff
@@ -11,5 +11,47 @@ class ApplicationController < Sinatra::Base
     erb :home
   end
 
+  get '/registrations/signup' do
+
+    erb :'/registrations/signup'
+  end
+
+  post '/registrations' do 
+    # use the data in params to create a new user and log them in by
+    # setting the session[:id] equal to the user's id here
+    
+    redirect '/users/home'
+  end
+
+  get '/sessions/login' do
+
+    # the line of code below render the view page in app/views/sessions/login.erb
+    erb :'sessions/login'
+  end
+
+  post '/sessions' do
+    # find the user who submitted the log in forms by looking in your database 
```

Comment:
I want to discourage students from thinking of comments as a place to put paragraphs of text. Can you remove thes?